### DISCUSSION TITLE
ACTIVATING AMT TO NEW MESHCENTRAL SERVER HELP!

### DISCUSSION THREAD

**Discussion by mtran-88** [Ylianst/MeshCentral#6023] [Q&A]
Hi, I am totally new to this AMT stuff, and would like to get details on how to activate the AMT to my new server.  We bought a bunch of NUCs that was using a management server set up by another entity.  The server is unreliable, and we do not see all of the NUC devices.  I was tasked to set up our own server to manage our NUCs.  All of the NUCs are AMT capable.  I just wondering what is needed for me to switch all of them to the new server.  I have been at this for a week, and no progress on the AMT side.  I don't even see the 'connect' button.  I have gone through the various guides, and still not getting anywhere with this.  Would someone please help me?  I am in desperate need of assistance on this.  This is crucial for the project moving forward.    I am technically capable, but this is giving me troubles, and it's getting frustrating.  I really appreciate any assistance on this.  I will volunteer to update the meshcentral document with updated details and gotchas for future new users like me.  ;OP

**Comment by si458** [comment]
First thing first, you need to reset the amt on the nucs,
I believe you can just unplug the batterys and that resets the amt? 
Then create a meshgroup and setup amt in its features,
Then install meshagent on all the nucs and add them to ur group,
Then pick a device and see if shows up as activated or no credentials or trying credentials, etc

**Reply by mtran-88** [reply]
Hi, I would have to open up all of the NUCs?  the majority is on site and at various locations.   Is there another way to reset that?  there is another way in the MEBx, but that did not really do a reset to factory.  Let me know your thoughts.  Thanks.

**Reply by si458** [reply]
if you know the intel AMT passwords for each of the units, check you can access the AMT UI http://localhost:16992 on the units, 
if you know the pass and success in the UI, you can remove the settings manually using the web ui, and then let meshcentral reconfig it in CCM mode
if you dont know the passwords then you will need physical access im afraid

**Reply by mtran-88** [reply]
I do have access to the AMT UI, but the only section that I think is relavent is this one.  What is the 'Shared FQDN' option do?
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/21a1d356-4322-44f6-a3a9-bbe840857221)

**Reply by si458** [reply]
what does the `system status` page look like?
the should be extra options like `internet settings` down the left hand side and urs is missing?

**Reply by si458** [reply]
sorry ignore me, the `internet settings` only appears IF you use the meshcentral AMT.
![image](https://github.com/Ylianst/MeshCentral/assets/765314/3774153a-11d0-40e7-be9a-9a9e60172d4e)
this is my output

**Reply by si458** [reply]
one thing you can try is download the `meshcmd.exe`
and then run `meshcmd.exe AmtDeactivate` (CCM mode)
or `meshcmd.exe AmtAcmDeactivate --user admin --pass pass --type full` (ACM mode)

**Reply by mtran-88** [reply]
Sorry, that just turns off CCM, and ACM mode.  How does that help?  At least I got the AMT 'Connect' Button now. Not sure what I did exactly.  :) 
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/198a1185-86d2-42b4-af62-d2fab74a4fb9)

**Reply by si458** [reply]
basically the commands will deactivate intel amt if its previously been activated (only if it knows the password or you give it it).

then when the computer restarts, because you have set the dns suffix already, meshcentral will notice this
and also because the intel amt is now deactivated, and you have the AMT setup in the groups, it then reactivated it for you!

just be aware, chances are the amt password might have changed! unless u specified a password in the group amt settings.

you can check the devices current amt password, by going to the `My Server Console` tab under `My Server` then typing amtpasswords

**Reply by si458** [reply]
but from the looks of the pic it works! 👍 
so you just need to replicate this to all the other units now :)

**Reply by mtran-88** [reply]
Hi, I am trying to replicate it on another NUC.  Not successful yet.  ;( Running into similar issues, but now it's CCM.   Also, I did not set the DNS suffix.  How would I change that to another DNS suffix?  
This is all I got...
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/d4e5971f-1bae-47d5-a1fc-0163161249d0)

**Reply by si458** [reply]
You only need the dns suffix if u want the device in acm mode if i remember correctly.

So again check u can access the amt Web ui, so u know what the user/pass is.

Once u know that, run the amtdeactivate with the `--user admin --pass pass` let it deactivate, then let it activate in meshcentral as ccm, then u can access the intel amt tab, then get into it amt via meshcentral to change the dns suffix, then deactivate/reactivate in acm mode

It really is a long game when it comes to not physically having access to the devices

**Reply by mtran-88** [reply]
Hi si458, I appreciate all your help!  You may have to spell it out for me on some of this since I am still somewhat not familiar with it.  :OP  does amt activate automatically after deactivating?  I used meshcmd and used amtconfig --user admin --pass password, and AMT CCM activated, but not seeing the 'Connect' button in the AMT tab.  

How would I change the Trusted DNS suffix in Meshcentral?  I will take what I learn and pass it forward.  Thanks again!  

![image](https://github.com/Ylianst/MeshCentral/assets/166559124/550b442a-9a32-4444-a886-7795191c970e)

C:\Windows\System32>meshcmd amtinfo
Intel AMT v15.0.35, pre-provisioning state.
Wired Enabled, DHCP, 48:21:0B:51:8F:01
Wireless Disabled, DHCP, 8C:17:59:7B:6A:92
Trusted DNS suffix: www.How.Do.I.Change.This.com
Connection Status: Outside, CIRA: Disconnected.

**Reply by si458** [reply]
Intel amt auto activate only happens if u have set it up in the groups settings page, can u share what ur group settings looks like?

**Reply by mtran-88** [reply]
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/f5c4a916-2124-4d7b-b35b-cb65b128169c)

**Reply by si458** [reply]
Yeh so `fully automatic` basically means if the device has amt, meshcentral will try to activate it into acm mode, if that's not possible, then try activate it into ccm mode, and if that is not possible, just leave it alone

**Reply by mtran-88** [reply]
Hi, it's activated in CCM right now.   It says it is pre-provisioning state. 
C:\Windows\System32>meshcmd amtinfo
Intel AMT v15.0.35, **pre-provisioning state.**
Wired Enabled, DHCP, 48:21:0B:51:8F:01
Wireless Disabled, DHCP, 8C:17:59:7B:6A:92
Trusted DNS suffix:  **www.How.Do.I.Change.This.com**
Connection Status: Outside, CIRA: Disconnected.
